modules/shBidAdapter.js

```diff
@@ -133,6 +147,14 @@ function createBids(bidRes, reqData) {
 
   bidRes.bids.forEach(function (bid) {
     const reqBid = bidMap[bid.bidId];
+    let currentBidRequest;
```

Comment:
This could be simplified to

```suggestion
    const currentBidRequest = reqBid.bidRequest.bids.find(bid2 => bid.bidId ===  bid2.bidId);
```

Reply:
Hi @muuki88! Changed :)

Reply:
Nice :sunglasses: